test/telnyx/network_test.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require_relative "../test_helper"
+
+module Telnyx
+  class NetworkTest < Test::Unit::TestCase
+    # should "be creatable" do
```

Comment:
We have commented all tests as don't have access to Telnyx-mock. So there might be any issue as these are untested.